.github/workflows/dependency-review-action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v4
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v4
+        with:
+	  license-check: false
```

Comment:
Shouldn't we enable the license check? Why is the check switched off?

Reply:
Because you will have to manually setup an allow or deny list of license names...